### PR TITLE
ReleaseNotes: Updated changelog and release notes for 8.0.0

### PR DESCRIPTION
**Pull request by grafanabot**
This exciting! So much has changed!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,39 @@
...
+* **Dashboard:** Fix issues where unsaved-changes warning is not displayed. [#34989](https://github.com/grafana/grafana/pull/34989), [@torkelo](https://github.com/torkelo)
+* **Login:** Fixes Unauthorized message showing when on login page or snapshot page. [#35311](https://github.com/grafana/grafana/pull/35311), [@torkelo](https://github.com/torkelo)
+* **NodeGraph:** Fix sorting markers in grid view. [#35200](https://github.com/grafana/grafana/pull/35200), [@aocenas](https://github.com/aocenas)
+* **Short URL:** Include orgId in generated short URLs. [#34696](https://github.com/grafana/grafana/pull/34696), [@farodin91](https://github.com/farodin91)
+* **Variables:** Support raw values of boolean type. [#34727](https://github.com/grafana/grafana/pull/34727), [@simPod](https://github.com/simPod)
+
+### Breaking changes
+
+
+The following endpoints were deprecated for Grafana v5.0 and support for them has now been removed:
```
Comment (dprokop) [inline CHANGELOG.md:28]: Grafana v.5.0 seems wrong to me! :)

Comment (dprokop) [reply CHANGELOG.md:28]: cc @Elfo404

Comment (Elfo404) [reply CHANGELOG.md:28]: @dsotirakis is it correct?

Comment (marefr) [reply CHANGELOG.md:28]: They were deprecated in v5.0 and are now removed in v8.0. Maybe could write that better?

**Comment by marefr** [comment]
I think we're missing an update of the release notes index page? 

Something to discuss in the future: Do we want to merge the release notes of beta's and stable into one, e.g. release notes for v8.0 includes beta1, 2, 3 and stable features and bugs together with a separate breaking change section of all breaking changes in v8.0? Think this would make it easier to get a full insight of what a stable release changes.

**Pull request closed without merging**